test-data/unit/check-classes.test

```diff
@@ -2051,6 +2051,16 @@ class X: pass
 foo(X)
 [out]
 
+[case testTypeUsingTypeCTypeAnyMember]
+from typing import Type, Any
+def foo(arg: Type[Any]):
+    x = arg.member_name
```

Comment:
I added the assignment in a new commit to the PR, if you want to accept it as-is.

My attempt to implement the fallback is in https://github.com/dmoisset/mypy/commit/b6d65489adfe3f68ddd2f254e3c6977afe9d3940 ... it is failing the tests as I mentioned (but appears to be working fine in some external test cases). I understand your explanation @JelleZijlstra , but I'm not certain how to fix the issue (i.e, what to do to get typeshed methods available in the test suite, and whether that is a good idea or if I should write the test in a different way).